src/UI/Component/Chart/Speedo/Factory.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/* Copyright (c) 2017 Ralph Dittrich <dittrich@qualitus.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Chart\Speedo;
+
+/**
+ * Interface Factory
+ * @package ILIAS\UI\Component\Chart\Speedo
+ */
+interface Factory {
+
+    /**
+     * ---
+     * description:
+     *   purpose: The standard speedo is used if the speedo is to be rendered in it's original size.
+     *

```

Comment:
composition is missing, proposal:
composition: The standard speedo is composed of one bar representing a score achieved in relation to some goal (maximum value) and a minimum indicated by some pointer. The diagnostic value is represented by a second bar bellow the first one.